applications/ConstitutiveLawsApplication/custom_constitutive/hyper_elastic_simo_taylor_neo_hookean_plane_strain_2d.h

```diff
@@ -0,0 +1,225 @@
+// KRATOS ___                _   _ _         _   _             __                       _
+//       / __\___  _ __  ___| |_(_) |_ _   _| |_(_)_   _____  / /  __ ___      _____   /_\  _ __  _ __
+//      / /  / _ \| '_ \/ __| __| | __| | | | __| \ \ / / _ \/ /  / _` \ \ /\ / / __| //_\\| '_ \| '_  |
+//     / /__| (_) | | | \__ \ |_| | |_| |_| | |_| |\ V /  __/ /__| (_| |\ V  V /\__ \/  _  \ |_) | |_) |
+//     \____/\___/|_| |_|___/\__|_|\__|\__,_|\__|_| \_/ \___\____/\__,_| \_/\_/ |___/\_/ \_/ .__/| .__/
+//                                                                                         |_|   |_|
+//
+//  License:         BSD License
+//                   license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+
+#pragma once
```

Comment:
Hmm what is this? :)

Reply:
A not standard way to do the 

#if !defined(NAME_H_INCLUDED )
#define  NAME_H_INCLUDED

Reply:
to be honest i am with   @loumalouomega on this. If it is not standard we shall avoid using it (it would be an enormous effort to add it at a later time)

Reply:
adding to  @KratosMultiphysics/technical-committee  for discussion

Reply:
OK. I'll remove it from this PR as we need to merge ASAP. After the KTC discussion we might (or not) revert the macro addition.

Reply:
The decision of @KratosMultiphysics/technical-committee is to start promoting the use of `pragma once`. One of the reasons of this is that it is already being used inside pybind, which is a hard compilation dependency. Besides, we think that this is a much user-developer solution for newbies.